ListingManager/Program.cs

```diff
@@ -22,6 +22,10 @@ public class Program
             bool byFolder = false,
             bool chapterOnly = false)
         {
+            while (path.EndsWith("\\"))
```

Comment:
This works (at least on Windows), but a cleaner solution would be to change the type of `path` to be a `DirectoryInfo` then you can let the default model binder deal with things like slashes for you.

Reply:
Makes sense, going to reference this in a new issue to do in the future